__tests__/app.test.js

```diff
@@ -9,6 +10,20 @@ const data = require("../db/data/test-data/index")
 beforeAll(() => seed(data));
 afterAll(() => db.end());
 
+describe("GET /api", () => {
+    test("200:should respond with an object describing all the available endpoints on the API ", () => {
+        return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({ body }) => {
+                fs.readFile('./endpoints.json', 'utf8')
```

Comment:
Although this works you have a nested .then which we try to avoid. There is no reason you can't just require in the file to the test suite and assert it is sent back by the endpoint in the test - without having to use fs in the test.

Reply:
Sorry didn't get what you mean here